src/components/TextFieldChip/TextFieldChips.test.tsx

```diff
@@ -138,6 +138,18 @@ describe('components/TextFieldChips', () => {
     expect(callbackOnInputChange).toHaveBeenCalledWith('')
   })
 
+  test('should add chip on blur when addOnBlur', async () => {
```

Comment:
would be nice to add a test for updating a test with blur

Reply:
And also, a test for `clearInputValue()`.

Conclusion : you should test all your `if` condition of your `handleClickAway` function

Reply:
Thanks, will tackle this Monday morning when I'm at my IDE 👍